test-userdata.yaml

```diff
@@ -81,3 +81,6 @@ sysctl:
 # Edit sysfs paths
 sysfs:
   /sys/kernel/mm/transparent_hugepage/enabled: never
+
+# Dummy Scalyr key
+scalyr_account_key: 123
```

Comment:
Should the key be an int? It seems wrong. I would assume it should be a string?

Reply:
Updated it to a string. But it shouldn't really have an effect in any way on the test.
The test itself just checked if scalyr_key is configured and checks if the config is properly set.